eng/scripts/TypeSpec-Generate-Sdk.ps1

```diff
@@ -0,0 +1,35 @@
+[CmdletBinding()]
+param (
+  [Parameter(Position = 0)]
+  [ValidateNotNullOrEmpty()]
+  [string] $SdkRepoRootDirectory,
```

Comment:
This looks like it will require a full path. Can we have some conventions that allow folks to just pass the language and then use that to map to a well know convention? 

For example if they pass "java" then we would walk to the root of the git clone and the look for a directory named "azure-sdk-for-java" and if found use that. In a future incarnation we could even do the clone there if we wanted.

Reply:
The cloned sdk repo and cloned spec repo local paths are arbitrary across users. We cannot assume user follow the convention to clone the repo locally.

Reply:
Yes they are but if we document and provide the error message to indicate what is the convention then we could still potentially use a shorthand of just the language, or they could pass the full path like you have. I believe @m-nash was suggesting trying to use the shorthand so that people didn't need to change their usages but suspect it wouldn't be that big of an issue either way and we can always improve if needed.

Reply:
Sounds good, we will enhance this later if needed.